### ISSUE TITLE
Minerals Production Reporter Handbook - combine into single PDF

### ISSUE THREAD

**Issue by cthomasONRR** [ONRR/onrr.gov-site#1213]
I need to:

- [x] Take all the 508-compliant PDF files and combine into a single PDF
- [x] Assess if the combined PDF is compliant
- [x] If not compliant, attempt to remediate
- [x] If not able to remediate, talk to Shannon H about making either a combined Word document or a .zip download of all the PDFs

**Comment by cthomasONRR** [comment]
Here is the combined PDF: [MPRH-AllDocs-Combined.pdf](https://github.com/ONRR/onrr.gov-site/files/7650695/MPRH-AllDocs-Combined.pdf)

Here is the email I sent to Shannon H that explains the issue: 

Hi Shannon,

Attached is the combined MPRH PDF that I created. It unfortunately looks like when you combine individual documents that pass the accessibility check, the resulting combined document will have new errors.

There are tag annotation issues with the hyperlinks in Ch 3 and alt text issues in Ch 6. Both of these individual PDFs do not show these errors. There is also that new table header issue that’s cropping up in every table within the document. I tried to resolve a few, but it will be very tedious and time-consuming to resolve all of them.

I’m curious if you think we need a combined MPRH PDF, or if you’re interested in discussing other options. One option would be to have a combined (locked) Word document, another option would be to have a .zip download that would include all of the individual compliant PDFs.

Let me know what you think! 
Thanks again for all your time and help with the 508 compliance efforts!

Christine Thomas

**Comment by cthomasONRR** [comment]
Shannon H responded with: 


> Good morning Christine, 
> 
> I greatly appreciate the alternate ideas, yet I feel the Production Handbook should display like the Minerals Revenue Reporter Handbook and the Solid Minerals Reporter Handbook on https://onrr.gov/reportpay/Handbooks/index.htm#production-handbook, with a complete handbook. I’ve cc’d Aaron Lindquist in case he has any additional input.
> 
> I will re-start my search for a compliant copy of the production handbook in RRM, I started and had to quit the search for a heat ticket installation. If I can’t find one I will use your copy and make it compliant, I believe I should be able to have it to you by the end of the week, depending on the issues I encounter and I’m assuming probably a very large document. 
> 
> I’m currently unable to get on Pulse, so this will delay my search for a possible RRM copy. I’ll keep you posted.
> 
> Shannon  H.
> 

@mcharg It looks like Shannon H will take responsibility for making this combined document 508 compliant, then send the compliant doc over to us to upload. I think this issue can be closed if you're ok with that. Thanks!

**Comment by mcharg** [comment]
@cthomasONRR I'm okay with that.
